README.md

```diff
@@ -2,7 +2,7 @@
 
 Secrets Store CSI driver for Kubernetes secrets - Integrates secrets stores with Kubernetes via a [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/) volume.
 
-The Secrets Store CSI driver `secrets-store.csi.k8s.com` allows Kubernetes to mount multiple secrets, keys, and certs stored in enterprise-grade external secrets stores into their pods as a volume. Once the Volume is attached, the data in it is mounted into the container's file system.
+The Secrets Store CSI driver `secrets-store.csi.x-k8s.com` allows Kubernetes to mount multiple secrets, keys, and certs stored in enterprise-grade external secrets stores into their pods as a volume. Once the Volume is attached, the data in it is mounted into the container's file system.
```

Comment:
@craiglpeters This is the driver name which has not yet been changed. The driver name will be updated after the repo move - https://github.com/deislabs/secrets-store-csi-driver/issues/129